scripts/rollupCreation.ts

```diff
@@ -339,7 +339,7 @@ async function _getDevRollupConfig(
     layerZeroSmallStepEdgeHeight: 2 ** 23,
     numBigStepLevel: 1,
     challengeGracePeriodBlocks: 10,
-    bufferConfig: { threshold: 600, max: 14400, replenishRateInBasis: 500 },
```

Comment:
why dont we just disable the delay proof why are we changing this?

Reply:
I don't understand what you meant. The config link above is where this PR is modifying. This config is not the type of this [handy config](https://github.com/EspressoSystems/nitro-contracts/blob/d988f19afb551fd0f9ec50e7bf3ac698119a5df7/scripts/boldUpgradeCommon.ts#L50), so I have to update `bufferConfig`.